migration/sql-files/068-limit-ooe-to-space.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR REPLACE FUNCTION limit_ooe_to_space() RETURNS void as $$
+-- limit_ooe_to_space() function limits order to space
+	DECLARE
+		i integer;
+		r RECORD;
+		a RECORD;
+		xyz CURSOR FOR SELECT id, execution_order, space_id from work_items;
+		abc CURSOR FOR SELECT id from spaces;
+	BEGIN
+		open abc;
+			FOR a in FETCH ALL FROM abc
+				LOOP
+					i:=1000;
+					open xyz;
+						FOR r in FETCH ALL FROM xyz
+							LOOP
+								UPDATE work_items set execution_order=i where id=r.id AND space_id=a.id;
+								i := i+1000;
```

Comment:
Could you please elaborate the logic here ? In each space, are we setting the execution order values to a multiple of 1000 ?